### PR TITLE
docs: replace some references of kaboom with kaplay

### PR DESCRIPTION
**Pull request by DaInfLoop**
I missed a few since I wasn't sure if it'd be a good idea to change them, although go ahead and edit it if you want

### CODE REVIEW

**Comment by amyspark-ng** [comment]
good

File: README.md [hunk 1]
```diff
@@ -101,10 +101,10 @@ onKeyDown("w", () => {
 
 ### Start a Project With `create-kaboom`
 
-The fastest way to start a Kaboom game is with [`create-kaboom`](https://github.com/replit/kaboom/tree/master/pkgs/create)
+The fastest way to start a KaPlay game is with [`create-kaplay`](https://github.com/marklovers/create-kaplay)
```
Comment (niceEli) [inline README.md]: https://kaplayjs.com/presskit/

File: README.md [hunk 2]
```diff
@@ -139,7 +139,7 @@ also works with CommonJS
 const kaboom = require("kaplay");
 ```
 
-Note that you'll need to use a bundler like `esbuild` or `webpack` to use Kaboom with NPM
+Note that you'll need to use a bundler like `esbuild` or `webpack` to use KaPlay with NPM
```
Comment (niceEli) [inline README.md]: also last comment here too

**Review by niceEli — CHANGES_REQUESTED** [review]
please fix branding 
https://kaplayjs.com/presskit/

**Review by niceEli — APPROVED** [review]
👍🏻

**Pull request merged**